### DISCUSSION TITLE
About the compound_id of the accounts table described in the documentation

### DISCUSSION THREAD

**Discussion by ken880guchi** [nextauthjs/next-auth#2495] [Help] [answered]
What value should I set for the compound_id column? If anyone knows, please let me know.
I don't know what it is used for. 🙏 

https://next-auth.js.org/adapters/typeorm/mysql

**Comment by StefanSelfTaught** [comment, marked as answer]
The `compound_id` is just a hash ganerated with node crypto module and it ensures that there is only one instance of an OAuth account in a way that works across different databases.

This is how it works with the default adapter:

```js
const compoundId = createHash("sha256")
  .update(`${providerId}:${providerAccountId}`)
  .digest("hex")

**Reply by ken880guchi** [reply]
@StefanSelfTaught thank you for the explanation, it is easy to understand.
